CondTools/HLT/src/AlCaRecoTriggerBitsRcdRead.cc

```diff
@@ -132,7 +132,7 @@ void AlCaRecoTriggerBitsRcdRead::beginRun(const edm::Run &run, const edm::EventS
 }
 
 ///////////////////////////////////////////////////////////////////////
-void AlCaRecoTriggerBitsRcdRead::endJob() {
+void AlCaRecoTriggerBitsRcdRead::endRun(edm::Run const &, edm::EventSetup const &) {
```

Comment:
@malbouis there's another mistake here. This should have not been changed to `endJob` as now this is called at every `endRun` transition instead that only once for the last examined IOV.
See a fix at https://github.com/cms-sw/cmssw/pull/36362.

Reply:
Sorry about that and thanks for submitting a fix!